presto-main/src/main/java/com/facebook/presto/resourcemanager/ResourceManagerClusterStateProvider.java

```diff
@@ -137,6 +140,10 @@ public ResourceManagerClusterStateProvider(
                 }
             }
         }, 100, 100, MILLISECONDS);
+
+        scheduledExecutorService.scheduleAtFixedRate(() -> {
+            adjustedQueueSize.set(computeAdjustedQueueSize());
+        }, 100, 1000, MILLISECONDS);
```

Comment:
Do we want this to run every second or every 100 milliseconds?

Reply:
I think 1 sec is sufficient for re-computing the adjusted queue size, as we don't expect "/v1/cluster" endpoint which exposes this to be called that frequently.